<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>802</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>휴먼둥근헤드라인</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;정렬데이터 입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="I">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>70</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>정렬</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(※ 띄어쓰기로 구분)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;수치입력&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;정렬결과&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insertion Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selection Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bubble Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quick Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Radix Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Merge Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Heap Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shell Sort&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="Insertionbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Mergebox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Selectionbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Bubblebox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Shellbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Quickbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Radixbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>650</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼고딕</family>
     </font>
    </property>
    <property name="text">
     <string>범위설정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>생성</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;수행시간&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>79</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="MinRange"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>휴먼고딕</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MaxRange"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>123</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>휴먼고딕</family>
        </font>
       </property>
       <property name="text">
        <string>개수설정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numCount"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="Timebox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>301</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>700</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>199</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="SelfCheck">
       <property name="font">
        <font>
         <family>휴먼고딕</family>
        </font>
       </property>
       <property name="text">
        <string>직접 입력</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RandomCheck">
       <property name="font">
        <font>
         <family>휴먼고딕</family>
        </font>
       </property>
       <property name="text">
        <string>랜덤 입력</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="Heapbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>401</width>
      <height>87</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>399</width>
       <height>85</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SelfCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MinRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelfCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MaxRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelfCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>numCount</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelfCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RandomCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MinRange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RandomCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MaxRange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RandomCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>numCount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RandomCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RandomCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>I</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1075</x>
     <y>745</y>
    </hint>
    <hint type="destinationlabel">
     <x>1071</x>
     <y>771</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelfCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>I</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
